docs/source/ecf/guidance.html.md

```diff
@@ -699,8 +699,100 @@ As soon as school induction tutors have entered the information to the DfE servi
 GET /api/v3/participants/ecf/transfers
 ```
 
+#### What providers will see in the API 
+
+<table class="govuk-table"> 
+<caption class="govuk-table__caption govuk-table__caption--m">What providers will see when a participant is leaving them</caption> 
```

Comment:
amend table headers to match https://design-system.service.gov.uk/components/table/